.github/workflows/tests.yaml

```diff
@@ -20,6 +20,11 @@ jobs:
         python-version: ["3.7", "3.8", "3.9"]
         # Cherry-pick test modules to split the overall runtime roughly in half
         partition: [ci1, not ci1]
+        include:
+          - partition: "ci1"
+            partition-label: "ci1"
+          - partition: "not ci1"
+            partition-label: "notci1"
```

Comment:
Not strictly necessary, but I didn't want the space from the pytest marker spec in the artifact name